src/c++/perf_analyzer/c_api_helpers/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
All these c_api_helpers files are part of triton_local so they should just live in there. Each subdirectory of client_backend is an actual client backend.